spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1369,16 +1363,23 @@ def _do_load(self):
         # algorithms needed for loading the binaries to the SpiNNaker machine
         optional_algorithms.append("FrontEndCommonLoadExecutableImages")
 
+        # Get the executable targets
+        optional_algorithms.append("FrontEndCommonGraphBinaryGatherer")
```

Comment:
dont mind here. as stated before. the position in the list was not what we were disucssing, more the move into do_load